test/dotnet-pack.Tests/PackTests.cs

```diff
@@ -80,15 +80,15 @@ public void SettingVersionSuffixFlag_ShouldStampAssemblyInfoInOutputAssemblyAndP
             outputPackage.Should().Exist();
         }
 
-        [Fact(Skip = "Test project missing")]
+        [Fact]
         public void HasIncludedFiles()
         {
             var testInstance = _testAssetsManager.CopyTestAsset("EndToEndTestApp")
                 .WithSource();
 
             new DotnetPackCommand(Log)
                 .WithWorkingDirectory(testInstance.Path)
-                .Execute()
+                .Execute("-c", "Debug")
```

Comment:
Is the -c Debug part necessary?

Reply:
By default, it appears to be the release path. Therefore, the debug configuration was used to ensure the correct file path.